searx/engines/duckduckgo_definitions.py

```diff
@@ -10,30 +10,57 @@
 """
```

Comment:
Could you please clean up the indentation in this file? The way dictionaries are indented are inconsistent and ugly. I know it is old code, but please take care of this. :)

```python
 results.append({
                'infobox': heading,
                'id': infobox_id,
                'content': content,
                'img_src': image,
                'attributes': attributes,
                'urls': urls,
                'relatedTopics': relatedTopics
                })
```
could be
```python
results.append({'infobox': heading,
                'id': infobox_id,
                'content': content,
                'img_src': image,
                'attributes': attributes,
                'urls': urls,
                'relatedTopics': relatedTopics})
```

Reply:
💯

Reply:
Done